### PR TITLE
add name_shorten() in Python

### PR DESCRIPTION
**Pull request by fikriazain**
Shorten a long name to easy recognize name of a person in your data. This is for hacktoberfest 2020

### CODE REVIEW

**Comment by jazibjafri** [comment]
@fikriazain remove the file that is not used and make sure to read the [readme](https://github.com/JazibJafri/hacktoberfest-2020/blob/main/README.md)

**Comment by fikriazain** [comment]
@JazibJafri done sir and i already read the readme file

File: src/python/fikriazain-name-shorten.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#  Shorten a name
+#  For example Michael Henderson --> Henderson, M.
+#  Hope you enjoy
+
+def name_shortener(user_name):
```
Comment (jazibjafri) [inline src/python/fikriazain-name-shorten.py]: function name should the same as in the filename

File: src/python/fikriazain-name-shorten.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	# Case if name have only 1 word
+	if (len(name_lst) > 1):
+		print(",", end = '')
+
+	for i in range(len(name_lst)-1):
+		# print first word with capital ' <huruf awal nama 1 kata dengan kapital>.'
+		print(' ' + name_lst[i][0].capitalize(), end = '.') 
+
+name_shortener(user_name)
```
Comment (jazibjafri) [inline src/python/fikriazain-name-shorten.py]: don't you think you should pass string here?

**Review by jazibjafri — CHANGES_REQUESTED** [review]
Some comments

File: src/python/fikriazain-name-shorten.py (hunk 1 above)
Comment (fikriazain) [reply src/python/fikriazain-name-shorten.py]: ohh i see okay sorry

File: src/python/fikriazain-name-shorten.py (hunk 2 above)
Comment (fikriazain) [reply src/python/fikriazain-name-shorten.py]: oh yeah sorry

**Comment by fikriazain** [comment]
@JazibJafri i already fix it

**Review by jazibjafri — APPROVED** [review]

**Pull request merged**